### PR TITLE
[bug] Cannot determine payment method when funding/deleting

### PR DESCRIPTION
**Pull request by nwithan8**
# Description

- Use payment method object type as fallback when determining type for fund, delete functions

# Testing

- Update, add unit test to confirm legacy (by ID prefix) and new (by object type) both work to determine payment method type

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

**Review by jchen293 — APPROVED** [review]

**Review by jchen293 — APPROVED** [review]

**Review by Justintime50 — CHANGES_REQUESTED** [review]
Sigh. Github lost my comment, let me rewrite it

File: easypost/services/billing_service.py [hunk 1]
```diff
@@ -66,9 +66,10 @@ def _get_payment_method_info(self, priority: str = "primary") -> List[str]:
 
         if payment_method_to_use and payment_methods[payment_method_to_use]:
             payment_method_id = payment_methods[payment_method_to_use]["id"]
-            if payment_method_id.startswith("card_"):
+            payment_method_object_type = payment_methods[payment_method_to_use].get("object", None)
```
[the remark this answers is not in the document]
Comment (nwithan8) [reply easypost/services/billing_service.py:69]: I'm not confident that our API always returns an "object" key.

[the remark this answers is not in the document]
Comment (Justintime50) [reply easypost/services/billing_service.py:69]: Then all of the other language PRs we've put up will fail as written. Please ask the necessary teams internally to gain confidence in the changes you are making before we move forward.

[the remark this answers is not in the document]
Comment (Justintime50) [reply easypost/services/billing_service.py:69]: I didn't realize that we are raising a `InvalidObjectError` below if we can't determine which endpoint to send it to - my bad. In this scenario, using `.get` here is fine for that reason.

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**